src/flamegraph/mod.rs

```diff
@@ -302,7 +323,7 @@ where
         )?;
 
         buffer.clear();
-        if href.is_some() {
+        if href_is_some {
```

Comment:
Can't this be `frame_attributes.href.is_some()`?

Reply:
I ran into an issue with conflicting borrowings. I chose to reduce the needed lifetime of `frame_attributes` to resolve it, by computing `href.is_some()` earlier.